.circleci/config.yml

```diff
@@ -29,35 +143,35 @@ workflows:
           requires:
             - fabric8_istio
 jobs:
+
+  build:
+    machine:
+      image: ubuntu-2204:2022.04.1
+    environment:
+      _JAVA_OPTIONS: "-Xms2g -Xmx2g"
+    steps:
+      - restore_main_cache
+      - checkout-code-then-restore-cache-then-attach-workspace
+      - build-project-then-save-docker-images
+      - persist-workspace-then-save-cache
+
+  fabric8_istio:
+    machine:
+      image: ubuntu-2204:2022.04.1
+    steps:
+      - checkout-code-then-restore-cache-then-attach-workspace
+      - build-fabric8-istio-integration-test
+
   test:
-    parallelism: 5 # parallel containers to split the tests among
+    parallelism: 10 # parallel containers to split the tests among
```

Comment:
I've changed this also, because we have fixed the incorrect caching problem - it brought an additional time to the build. Adding 5 more instances, shaves around 8-10 min from the build.